src/Strategy/DateTimeFormatterStrategy.php

```diff
@@ -86,18 +101,27 @@ public function extract($value)
      * @param mixed|string $value
      * @return mixed|DateTimeInterface
      */
-    public function hydrate($value)
+    public function hydrate($value, ?array $data = null)
     {
-        if ($value === '' || $value === null) {
-            return;
+        if ($value === '' || $value === null || $value instanceof DateTimeInterface) {
+            return $value;
+        }
+
+        if (! is_string($value)) {
+            throw new Exception\InvalidArgumentException(sprintf(
```

Comment:
We are missing `@throws` tag in PHPDoc of the method, as the exception is thrown here.